### ISSUE TITLE
Black Market Idea

### ISSUE THREAD

**Issue by Zironic** [Arx-Game/arxcode#76]
While we're waiting for contraband to be a thing, it would be interesting to give the black market a bonus to haggle while making it charge people fame (like with nobles trying to haggle with the normal market).

This would allow people who are fine with having a bad reputation find things cheaper.

**Comment by rozzingit** [comment]
Drop off the Economics option and make people have to roll Streetwise for it!

**Comment by TehomCD** [comment]
Contraband is kind of a thing - some items are marked as 'nosell' in their aquisitions modifiers, and there's a few drugs that way. We just don't presently have a way to distinguish a black market, though I believe something was set up in the crime branch. We could probably cherry pick that over at some point.

**Comment by Zironic** [comment]
Right now the black market is literally just the one room isn't it? So you'd just check if the caller is in that room.

Economics could be replaced by Legergemain

**Comment by TehomCD** [comment]
Yeah, sorry, I was mostly just thinking about the ways it differed from the crime implementation. It didn't use a physical market at all - there's a fence command where individual characters essentially act as the black market for one another, and you could use criminal resources acquired from crime to buy stuff rather than regular currency. But we could just change the market command and have it check for a "black market" tag on the room or whatever for having alternate behavior, sure.

**Comment by Zironic** [comment]
Yeah. I was thinking it would be a small cool thing that the crime players could enjoy while they wait for the much more ambitious crime system.

**Comment by TehomCD** [comment]
Yeah, thinking about it, it'd be a bad idea to allow contraband to be sold at all right now when there aren't recipes to use it, since people would be confused by that. I do like the bonus to haggle while having some fame penalty - maybe automatically applying the 'shady' propriety mod if they don't already have it.

**Comment by Precisi** [comment]
> 
> 
> Contraband is kind of a thing - some items are marked as 'nosell' in their aquisitions modifiers, and there's a few drugs that way. We just don't presently have a way to distinguish a black market, though I believe something was set up in the crime branch. We could probably cherry pick that over at some point.

As far as I know, the drugs are in the system, but we can't actually buy/do anything with them? 

I do like the idea of having some different options for the black market and I think the crime players would like that. People could even be brokers for others that don't want to get their hands dirty, etc.
